tests/drivers/spi/spi_loopback/boards/rpi_pico.conf

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2022 TOKITA Hiroshi <tokita.hiroshi@fujitsu.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# enable rpi_pico spi instance for loopback test
+CONFIG_SPI_LOOPBACK_DRV_NAME="SPI_0"
```

Comment:
Seen reworked PR for spi_loopback: https://github.com/zephyrproject-rtos/zephyr/pull/48222

Reply:
Hi, @galak, 

I'll prepare to apply the #48222 change to this PR.

I also failed #48222 same as @erwango observed.

@erwango mentioned in #48222

> @galak, as expected this test failed on our bench since #48358 was merged. Out of curiosity I launched a campaign using current PR and I still see issues so most likely it requires a bit more work. Do you have the same status on your side, or do you want me to dig a little bit more ?

I'm checking #48222 progress, and I'll report when the update is complete.

Thank you.

Reply:
Hi, @galak 

#48222 is correctly worked with also rpi_pico now.
I prepare to update this PR.

Reply:
Rebase done.